Gemfile.lock

```diff
@@ -105,7 +105,6 @@ PATH
 
 GEM
   remote: https://rubygems.org/
-  remote: https://enterprise.contribsys.com/
```

Comment:
The changes to Gemfile.lock should be reverted. To avoid accindetally making these changes in the future see https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/platform/engineering/sidekiq-enterprise-setup.md